.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Question/Support/Other
+about: If you have a usage question
+labels: Community support
+# This template is based on the scikit-learn template
+# https://github.com/scikit-learn/scikit-learn/blob/8e61534f1087703f476414d8dbd3688282f8eebf/.github/ISSUE_TEMPLATE/usage_question.md
```

Comment:
Going back and forth on whether this needs to be in here...

Reply:
It won't show up when someone makes a new issue as it is a comment in the yaml frontmatter rather than in the markdown body. See example here: https://github.com/ianhi/git-expt/issues/new?labels=Question&template=test.md and the source file https://raw.githubusercontent.com/ianhi/git-expt/master/.github/ISSUE_TEMPLATE/test.md

Reply:
thanks for the clarification.